packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorMenuItemTests.cs

```diff
@@ -0,0 +1,22 @@
+using Bunit;
+using Xunit;
+
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for <see cref="NimbleAnchorMenuItem"/>.
+/// </summary>
+public class NimbleAnchorMenuItemTests
+{
+    [Fact]
+    public void NimbleAnchorMenuItem_Render_HasAnchorMenuItemMarkup()
```

Comment:
Any reason to not follow other test examples (like the [Combobox](https://github.com/ni/nimble/blob/f60dedf600147b16916f876b749a4e019c2a6308/packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleComboboxTests.cs#L11)), and test _all_ attributes? I think the components that don't do that are some of the earlier ones.

Reply:
No problem. Tests added.